src/command/toggleattributecommand.js

```diff
@@ -30,43 +31,29 @@ export default class ToggleAttributeCommand extends Command {
 		/**
 		 * Attribute that will be set by the command.
 		 *
+		 * @readonly
 		 * @member {String}
 		 */
 		this.attributeKey = attributeKey;
 
 		/**
```

Comment:
Is this description still true? It's a bit convoluted and it hides the fact that we use document selection. Maybe we should write it directly that the value is set depending on document's selection.